lib/bin/lwm2m_carrier/doc/requirements.rst

```diff
@@ -25,13 +25,14 @@ Below are some of the requirements and limitations of the application while runn
 * The LwM2M carrier library controls the LTE link.
 
    * This is needed for storing keys to the modem, which requires disconnecting from an LTE link and connecting to it.
-   * The application should not connect to the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` event and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
+   * The application cannot use the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` event and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
+     Instead, the application must wait for the :c:macro:`LWM2M_CARRIER_EVENT_LTE_READY` event.
 
 * The LwM2M carrier library uses a TLS session for FOTA.
   TLS handshakes performed by the application might fail if the LwM2M carrier library is performing one at the same time.
 
    * If the application is in a TLS session and a :c:macro:`LWM2M_CARRIER_EVENT_FOTA_START` event is sent to the application, the application must immediately end the TLS session.
-   * TLS becomes available again upon the next :c:macro:`LWM2M_CARRIER_EVENT_READY` or :c:macro:`LWM2M_CARRIER_EVENT_DEFERRED` event.
+   * TLS becomes available again upon the next :c:macro:`LWM2M_CARRIER_EVENT_LTE_READY` event.
```

Comment:
Let's add a line here specifying the application must also close the DFU socket here, if in use.